packages/astro/test/dev-routing.test.js

```diff
@@ -151,9 +151,9 @@ describe('Development Routing', () => {
 			expect(response.status).to.equal(200);
 		});
 
-		it('200 when loading subpath root without trailing slash', async () => {
+		it('404 when loading subpath root without trailing slash', async () => {
```

Comment:
This is an example of previous behavior caused by a side-effect of what you set `buildOptions.site` to. 

The updated test better matches expected behavior.

Reply:
I thought we fixed this in a previous PR. Or was that only for `astro preview`?

Reply:
This is the one: https://github.com/withastro/astro/pull/2338#issue-1096656090

> @matthewp and I discussed `buildOptions.site` and agreed it represents a directory.

I’m curious how we ended up doing this again. That’s why I asked if the prior effort only fixed `astro preview`.

Reply:
I'm not sure!